src/components/pages/devices/devicesGrid/devicesConfig.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import moment from 'moment';
+import Config from 'app.config';
+import { IsSimulatedRenderer, ConnectionStatusRenderer } from 'components/shared/cellRenderers';
+import { EMPTY_FIELD_VAL, DEFAULT_TIME_FORMAT, gridValueFormatters } from 'components/shared/pcsGrid/pcsGridConfig';
+
+const { checkForEmpty } = gridValueFormatters;
+
+export const checkboxParams = {
+  headerCheckboxSelection: true,
+  headerCheckboxSelectionFilteredOnly: true,
+  checkboxSelection: true
+};
+
+/** A collection of column definitions for the devices grid */
+export const deviceColumnDefs = {
+  id: {
+    headerName: 'Device Name',
```

Comment:
Do we need to localize this? Should this be hard coded? Same comment for other places